### ISSUE TITLE
TimeLimit attribute

### ISSUE THREAD

**Issue by rlinfati** [COPT-Public/COPT.jl#15]
```
    m = JuMP.Model()
[...]
    JuMP.set_optimizer(m, COPT.Optimizer)
    JuMP.set_optimizer_attribute(m, "TimeLimit", 3600.0)
    JuMP.optimize!(m)
```

the optimizer do not stop after the time limit.
COPT.jl v1.0.1
Cardinal Optimizer v5.0.1. Build date Jun 20 2022
Setting parameter 'TimeLimit' to 3600

**Comment by COPT-Public** [comment]
Sorry for the slow reply, we will try to reproduce the issue and see how to fix it.

**Comment by lschork2** [comment]
Thank you for reporting the issue. This was a bug in COPT.jl, which is fixed now in version 1.0.2.
